### PR TITLE
react form with example and link added

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Issue
https://github.com/feimosi/frameworks-code-comparison/issues/3

#### Code snippet
```js
import React from 'react'

export default class ReactForm extends React.Component {
  constructor(props) {
    super(props)
    this.state = {
      formValue: ''
    }
  }

  handleChange = event => {
    this.setState({
      formValue: event.target.value
    })
  }

  render() {
    return (
      <form>
        <input
          value={this.state.formValue}
          placeholder="Enter value"
          onChange={this.handleChange}
        />
      </form>
    )
  }
}
```

### CODE REVIEW

**Comment by M-ZubairAhmed** [comment]
Please review and let me know if you would like some changes to be made to this PR

File: README.md [hunk 1]
```diff
@@ -829,6 +829,42 @@ export class ReactiveFormComponent implements OnInit {
 }
 ```
 
+### React
+
+React controlled components naturally keep some internal mutable state and update it based on user input via `setState()`.
```
Comment (feimosi) [inline README.md]: Could you write that there are two available approaches to forms: using controlled and uncontrolled inputs? And that the former is recommended.

File: README.md [hunk 2]
```diff
@@ -829,6 +829,42 @@ export class ReactiveFormComponent implements OnInit {
...
+
+```js
+import React from 'react'
+
+export default class ReactForm extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      formValue: ''
```
Comment (feimosi) [inline README.md]: Make it as some more real-life example - based on Angular example with `name` and `email` inputs.

**Review by feimosi — CHANGES_REQUESTED** [review]
Thanks! Take a look at my comments.

**Comment by M-ZubairAhmed** [comment]
Changes are made to the PR

**Comment by feimosi** [comment]
Looks good :+1:

**Pull request merged**